manifests/aggregator.pp

```diff
@@ -19,7 +19,7 @@
 
   file { "${telegraf::config_folder}/${name}.conf":
     ensure  => $_ensure,
-    content => inline_template("<%= require 'toml-rb'; TomlRB.dump({'aggregators'=>{'${plugin_type}'=>@options}}) %>"),
+    content => to_toml({ 'aggregators'=> { $plugin_type=> $options } }),
```

Comment:
```
steve@arch ~> puppet apply -e '$a = to_toml({'a' => 'b'})'
Warning: This function is deprecated, please use stdlib::to_toml instead. at ["unknown", 1]:["unknown", 0]
   (location: /home/steve/.puppetlabs/etc/code/modules/stdlib/lib/puppet/functions/deprecation.rb:35:in `deprecation')
Notice: Compiled catalog for arch.lan in environment production in 0.02 seconds
Notice: Applied catalog in 0.02 seconds
```
However you may have done on purpose to avoid the stdlib bump to v9  at this time.

Reply:
Indeed, because puppet/archive doesn't allow stdlib 9. We've a bit of a dependency problem on most modules that we need to sort out.

Reply:
I released archive and we can now depend on stdlib 9 (I hope)